go/vt/discovery/keyspace_events.go

```diff
@@ -252,16 +276,25 @@ func (kss *keyspaceState) ensureConsistentLocked() {
 		}
 	}
 
+	// clone the current moveTablesState, if any, to handle race conditions where it can get updated while we're broadcasting
```

Comment:
I know that it's awfully nitty, and a matter of opinion/preference, but I'd like for us to consistently use sentence capitalization and punctuation for new comments.